Packs/AWS-SQS/Integrations/AWS-SQS/AWS-SQS.py

```diff
@@ -33,30 +45,69 @@ def aws_session(service='sqs', region=None, roleArn=None, roleSessionName=None,
     elif AWS_rolePolicy is not None:
         kwargs.update({'Policy': AWS_rolePolicy})
 
-    if kwargs:
-        sts_client = boto3.client('sts')
+    if kwargs and not AWS_ACCESS_KEY_ID:
```

Comment:
can you inline comment for the different auth cases? i'm not sure I can understand where the old case is where a user does not configure `AWS_ACCESS_KEY_ID` nor `AWS_SECRET_ACCESS_KEY`.